Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -511,6 +511,66 @@ extension MapboxMap: AttributionDataSource {
     }
 }
 
+// MARK: - Feature State -
+
+extension MapboxMap {
+
+    /// Update the state map of a feature within a style source.
+    /// Update entries in the state map of a given feature within a style source. Only entries listed in the state map
+    /// will be updated. An entry in the feature state map that is not listed in `state` will retain its previous value.
+    ///
+    /// - Note: updates to feature state are asynchronous, so changes made by this method migth
+    ///         not be immediately visible using getStateFeature().
```

Comment:
Should I be concerned that there's no completion block or way to see when the updates are complete?

Reply:
The docs on this were incorrect. This is a synchronous call on iOS.